autogen/main/variables.tf.tmpl

```diff
@@ -979,6 +979,12 @@ variable "enable_l4_ilb_subsetting" {
   description = "Enable L4 ILB Subsetting on the cluster"
   default     = false
 }
+
+variable "disable_l4_lb_firewall_reconciliation" {
+  type        = bool
+  description = "Disable L4 Load Balancer firewall reconciliation"
+  default     = false
```

Comment:
Is it possible to use `null`?

Reply:
will check

Reply:
CI is working again, just waiting on this item.

Reply:
hopefully will get to this next day or so